### PR TITLE
Create helios-community-WL

### PR DESCRIPTION
**Pull request by KevinLydon**
Transfer Whitelist request for the quarterly Helios Community LKMEX distribution.

### CODE REVIEW

**Comment by PABourdais** [comment]
They will not accept it since you don't use a SC for LKMEX distribution

File: LKMEX-whitelist/helios-mitya-NFT-WL [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+erd1w3c0lqy8af6v27pq4qzn87t9glr33fk9qyym6mkye07gsn5xyanstvm0d9
+
+Purpose: 
+
+The Mitya NFT is designed as a reward for holders of the NFT through yeilds, and as an incentive to continue staking with Helios Staking. Royalities are added to farms to incease the rewards that are done every 41 days. LKMEX is one of these and we use a python script from the address below to distribute. 
+
+Wallet Address: 
+
+erd1w3c0lqy8af6v27pq4qzn87t9glr33fk9qyym6mkye07gsn5xyanstvm0d9
```
Comment (sasurobert) [inline LKMEX-whitelist/helios-mitya-NFT-WL:20]: We do not whitelist user addresses. Only SCs with well defined functions.

File: LKMEX-whitelist/helios-community-WL [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Helios Staking rewards its long term stakers with a quarterly distribution of LKMEX. Anyone who has at least 1 EGLD staked with Helios Staking as of 01/01/22 is eligible for the quarterly reward, which is farmed using a reserves wallet that was created as a fund designed to create more community initiatives. This wallet takes 5% of all Helios Staking staking commisions and farms and distribute from the wallet address erd1w69k99shct7gdmalf7t3n9qedsv2cnn04lrc249trphydnr273dskvlnpp
+
+Purpose: 
+
+The community quarterly distribution is a reward for long term Helios Staking delegators as well as being an incentive to continue staking.WE use a python script from the address below to distribute the LKMEX. 
+
+Wallet Address: 
+
+erd1w69k99shct7gdmalf7t3n9qedsv2cnn04lrc249trphydnr273dskvlnpp
```
Comment (sasurobert) [inline LKMEX-whitelist/helios-community-WL:15]: We do not whitelist user addresses. Only SCs with well defined functions.

**Review by sasurobert — CHANGES_REQUESTED** [review]

**Comment by sasurobert** [comment]
Hello Helios Community,
After a thorough analysis of your Pull Request, we have concluded that we are unable to provide a ‘whitelisted’ status. The reasoning behind this decision is the fact that your project uses ‘wallets’ to send arbitrary amounts of XMEX and should it ever happen that the project’s keys are compromised or some illegitimate XMEX transfers are made, it would be difficult to detect and take action.
Our advice would be to create a proper NFT Staking Smart Contract where only the owner is allowed to deposit XMEX. Users can claim their rewards if they stake the NFT or by sending the NFT to the contract (thus proving that they own the NFT).
Have a productive day!

**Pull request closed without merging**